Intersect (Core)/Config/Options.cs

```diff
@@ -135,6 +137,10 @@ public class Options
 
         public static int MinChatInterval => Instance.ChatOpts.MinIntervalBetweenChats;
 
+        public static bool ShowUnownedItems => Instance.LootOpts.ShowUnownedItems;
```

Comment:
Honestly I think it should be `public static LootOpts Loot => Instance.LootOpts` instead of adding more variables at the top level here.